Firebase/Messaging/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 2015-07-10 -- v3.0.3
+- Clean up unused `FIRMessaging_PROBER` macro.
+- Updated topic subscription warning.
+- Add a feature that queues up the topic requests when token is not available and syncs the topic when the token is generated.
```

Comment:
Fixed an issue that client should suspend the topic requests when token is not available and resume the topic operation when the token is generated.

And move this to the top.